devtools/common/src/main/resources/templates/basic-rest/java/build.gradle-template.ftl

```diff
@@ -35,5 +52,22 @@ compileJava {
     options.compilerArgs << '-parameters'
 }
 
+test {
+    useJUnitPlatform()
+}
+
+task testNative(type: Test) {
+    useJUnitPlatform()
+    description = 'Runs native image tests'
+    group = 'verification'
+
+    testClassesDirs = sourceSets.nativeTest.output.classesDirs
+    classpath = sourceSets.nativeTest.runtimeClasspath
+    shouldRunAfter test
+}
+
+testNative.dependsOn buildNative
```

Comment:
I suspect ideally such "dependsOn" links could be made implicit?

Reply:
What do you have in mind? I am sure you have some cool Gradle example I am missing 😉

Reply:
Actually it might make sense to have this task provided by the plugin itself. WDYT?

Reply:
Yes that's what I mean. Sorry actually not having an example at hand, I'm a bit rusty on plugin development (and Gradle 5 updated many APIs, haven't used it yet).

But conceptually, such things should be setup by the plugin. People don't need to know, so they won't screw it up ;)

Reply:
Makes sense! I'll check it out 👍

Reply:
Everything has now been moved into the plugin.